kube-derive/src/lib.rs

```diff
@@ -150,6 +150,18 @@ mod custom_resource;
 /// }
 /// ```
 ///
+/// ## Customizing Schemas
+/// Should you need to customize the schemas, you can use:
+/// - [Serde/Schemars Attributes](https://graham.cool/schemars/examples/3-schemars_attrs/) (no need to duplicate serde renames)
+/// - [`#[schemars(schema_with = "func")]`](https://graham.cool/schemars/examples/7-custom_serialization/) (e.g. like in the [`crd_derive` example](https://github.com/clux/kube-rs/blob/master/examples/crd_derive.rs))
```

Comment:
Or implement JsonSchema for own type or newtype around external type. This can be better if the type is used often.

Reply:
I'll add that with a link to your example comment: https://github.com/clux/kube-rs/issues/129#issuecomment-750852916